internal/connect/command_offset_describe.go

```diff
@@ -0,0 +1,136 @@
+package connect
+
+import (
+	"encoding/json"
+
+	connectv1 "github.com/confluentinc/ccloud-sdk-go-v2/connect/v1"
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+	"github.com/confluentinc/cli/v3/pkg/kafka"
+	"github.com/confluentinc/cli/v3/pkg/output"
+)
+
+type OffsetConnectorOut struct {
+	Id       string `human:"ID" json:"id" yaml:"id"`
+	Name     string `human:"Name" json:"name" yaml:"name"`
+	Offsets  string `human:"Offsets" json:"offsets,omitempty" yaml:"offsets,omitempty"`
+	Metadata string `human:"Metadata" json:"metadata,omitempty" yaml:"metadata,omitempty"`
+}
+
+type SerializedOffsetConnectorOut struct {
+	Id       string                                      `human:"ID" json:"id" yaml:"id"`
+	Name     string                                      `human:"Name" json:"name" yaml:"name"`
+	Offsets  []map[string]any                            `human:"Offsets" json:"offsets,omitempty" yaml:"offsets,omitempty"`
+	Metadata connectv1.ConnectV1ConnectorOffsetsMetadata `human:"Metadata" json:"metadata,omitempty" yaml:"metadata,omitempty"`
+}
+
+func (c *offsetCommand) newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "describe <id>",
+		Short:             "Describe connector offsets",
+		Args:              cobra.ExactArgs(1),
+		RunE:              c.getOffset,
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		Annotations:       map[string]string{pcmd.RunRequirement: pcmd.RequireNonAPIKeyCloudLogin},
```

Comment:
Please move this annotation to the root `confluent connect offset` command so we don't need to add it to every child command.

Reply:
done